### PR TITLE
Started a new runbook to explain the process of deploying a new MITx environment

### PR DESCRIPTION
**Pull request by blarghmatey**
There may still be some missing steps. Comments/suggestions welcome

### CODE REVIEW

File: docs/runbooks/build_new_mitx_env.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+In order to provision a new MITx environment the following steps are necessary:
+
+- Define the environment in the `environments.yml` file in the pillar repository
```
Comment (shaidar) [inline docs/runbooks/build_new_mitx_env.md]: Isn't this environment_settings.yml?

File: docs/runbooks/build_new_mitx_env.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  `VPC_NAME='My VPC' ENVIRONMENT=my-vpc BUSINESS_UNIT=residential salt-run state.orchestrate orchestrate.edx.mysql_schemas`
+- Deploy the edX app, worker, and xqueue-watcher instances
+  ```
+  export VPC_NAME='My VPC'
+  export ENVIRONMENT=my-vpc
+  export BUSINESS_UNIT=residential
+  export PURPOSE_PREFIX=residential
+  salt-run state.orchestrate orchestrate.edx
+  salt-run state.orchestrate orchestrate.edx
+  ```
```
Comment (shaidar) [inline docs/runbooks/build_new_mitx_env.md]: Any specific reason why we're exporting the three variables (VPC_NAME, ENVIRONMENT, BUSINESS_UNIT) multiple times? Is it mainly to ensure that they're always defined as opposed to possibly using the default values especially when the orchestration of different instances might not all happen at the same time?

File: docs/runbooks/build_new_mitx_env.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+In order to provision a new MITx environment the following steps are necessary:
+
+- Define the environment in the `environments.yml` file in the pillar repository
+- Provision the VPC
+  `VPC_NAME='My VPC' ENVIRONMENT=my-vpc BUSINESS_UNIT=residential salt-run state.orchestrate orchestrate.aws.mitx`
+- Accept the VPC peering connection via the AWS console
+- Update the route tables in the operations VPC to allow traffic from the new environment
+- Update the operations security groups for Consul and FluentD to allow traffic from the new environment
```
Comment (shaidar) [inline docs/runbooks/build_new_mitx_env.md]: Are there issues created for those manual steps above to possibly automate them?

File: docs/runbooks/build_new_mitx_env.md (hunk 1 above)
Comment (blarghmatey) [inline docs/runbooks/build_new_mitx_env.md]: Yes, I'll fix that

File: docs/runbooks/build_new_mitx_env.md (hunk 2 above)
Comment (blarghmatey) [inline docs/runbooks/build_new_mitx_env.md]: Just being explicit. If it is all run in one session then the exports can be executed just once.

File: docs/runbooks/build_new_mitx_env.md (hunk 3 above)
Comment (blarghmatey) [inline docs/runbooks/build_new_mitx_env.md]: No, not yet.

**Pull request merged**